VENTokenField/VENTokenField.m

```diff
@@ -86,10 +85,12 @@ - (void)setUpInit
     self.horizontalInset = VENTokenFieldDefaultHorizontalInset;
     self.tokenPadding = VENTokenFieldDefaultTokenPadding;
     self.minInputWidth = VENTokenFieldDefaultMinInputWidth;
+    self.toLabelPadding = VENTokenFieldDefaultToLabelPadding;
     self.colorScheme = [UIColor blueColor];
     self.toLabelTextColor = [UIColor colorWithRed:112/255.0f green:124/255.0f blue:124/255.0f alpha:1.0f];
     self.inputTextFieldTextColor = [UIColor colorWithRed:38/255.0f green:39/255.0f blue:41/255.0f alpha:1.0f];
-    
+    self.font = [UIFont fontWithName:@"HelveticaNeue" size:15.5];
```

Comment:
It's just a default value, consistent with the rest of the plugin. It'll be overriden by the element that will use this token field anyway ;)